### ISSUE TITLE
a11y: speaker avatar images are missing alt tags on session

### ISSUE THREAD

**Issue by coliff** [GoogleChrome/devsummit#313]
there was a previous issue regarding alt text (#246) that is closed... so thought it was worth opening a new issue for this.

e.g. https://developer.chrome.com/devsummit/sessions/next-generation-web-styling/

**Comment by jakearchibald** [comment]
Fixed! Thanks for reporting
